src/main/webapp/app/complaints/complaint.service.ts

```diff
@@ -249,7 +249,7 @@ export class ComplaintService implements IComplaintService {
             return undefined;
         }
 
-        return complaintStartDate!.add(course.maxComplaintTimeDays!, 'days');
+        return dayjs(complaintStartDate).add(course.maxComplaintTimeDays!, 'days');
```

Comment:
that's a good workaround and bugfix (and we should keep it), but we should make sure that all dates are already converted directly after they are downloaded from the server in the corresponding method that executes the http call.